### PR TITLE
Defect/de2453 site and phonenumber issues

### PR DESCRIPTION
**Pull request by silbermm**

### CODE REVIEW

File: Gateway/MinistryPlatform.Translation.Test/Services/CongregationRepositoryTest.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
```
Comment (andycanterbury) [inline Gateway/MinistryPlatform.Translation.Test/Services/CongregationRepositoryTest.cs]: Did you figure out a way to multi-thread unit tests? 
...or do you need to clean up your usings?

File: Gateway/MinistryPlatform.Translation.Test/Services/MinistryPlatformRestServiceIntegrationTest.cs [hunk 2]
```diff
@@ -491,6 +491,13 @@ public void GetEventParticipant()
             var participants = _fixture.UsingAuthenticationToken(_authToken).Search<MpEventParticipant>(filter, columns);
         }
 
+        [Test]
+        public void findCongregation()
```
Comment (andycanterbury) [inline Gateway/MinistryPlatform.Translation.Test/Services/MinistryPlatformRestServiceIntegrationTest.cs]: This test has no assertions. What is it testing?

File: CI/SQL/20161214_101033_DE2453_create_new_form_filed_ID_crossroadssite.sql [hunk 3]
```diff
@@ -0,0 +1,13 @@
+USE MinistryPlatform
+GO
+
+DECLARE @FormId INT = 28
+DECLARE @FieldId INT = 1517
```
Comment (silbermm) [inline CI/SQL/20161214_101033_DE2453_create_new_form_filed_ID_crossroadssite.sql]: Yes, already checked!

File: Gateway/MinistryPlatform.Translation.Test/Services/MinistryPlatformRestServiceIntegrationTest.cs (hunk 2 above)
Comment (silbermm) [inline Gateway/MinistryPlatform.Translation.Test/Services/MinistryPlatformRestServiceIntegrationTest.cs]: just an integration test to make sure permissions were correct and I used the correct search string

**Pull request merged**